Controllers/UsersController.cs

```diff
@@ -0,0 +1,51 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Taskmanager.Controllers.InputModels;
+using Taskmanager.Controllers.ViewModels;
+using Taskmanager.Data.Entities;
+using Taskmanager.Services.Interfaces;
+
+namespace Taskmanager.Controllers
+{
+    [ApiController]
+    [Route("api/users")]
+    public class UsersController : ControllerBase
+    {
+        private readonly IUserService _userService;
+
+        public UsersController(IUserService userService)
+        {
+            _userService = userService;
+        }
+
+        [HttpGet("{userId}")]
+        public async Task<ActionResult<UserViewModel>> GetOneByIdAsync([FromRoute] int userId)
+        {
+            User requestedUser = await _userService.GetByIdAsync(userId);
+
+            UserViewModel userToBeDisplayed = UserViewModel.MapUser(requestedUser);
```

Comment:
Naming is inconsistent, for todo lists it was todolistViewModel, but here it's userToBeDisplayed for some reason